### PR TITLE
Release 0.8.0

### PR DESCRIPTION
**Pull request by phoracek**
- adds nmstate component
- fixes kubemacpool bug that was failing re-applying of operator CR

### CODE REVIEW

**Comment by phoracek** [comment]
@SchSeba @qinqon could you please review?

File: manifests/cluster-network-addons/0.8.0/cluster-network-addons-operator.0.8.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+          },
+          "spec": {
+            "multus":{},
+            "linuxBridge":{},
+            "sriov":{},
+            "kubeMacPool": {
+              "rangeStart": "02:00:00:00:00:00",
+              "rangeEnd": "FD:FF:FF:FF:FF:FF"
+            },
+            "imagePullPolicy": "Always"
```
Comment (qinqon) [inline manifests/cluster-network-addons/0.8.0/cluster-network-addons-operator.0.8.0.clusterserviceversion.yaml]: nmstate missing

File: manifests/cluster-network-addons/0.8.0/cluster-network-addons-operator.0.8.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+  description: Deploy additional networking components for Kubernetes
+  keywords:
+    - KubeVirt
+    - Virtualization
+    - Networking
+    - Multus
+    - CNI
+    - macpool
+    - SR-IOV
+    - Bridge
```
Comment (qinqon) [inline manifests/cluster-network-addons/0.8.0/cluster-network-addons-operator.0.8.0.clusterserviceversion.yaml]: ditto

File: manifests/cluster-network-addons/0.8.0/cluster-network-addons-operator.0.8.0.clusterserviceversion.yaml (hunk 1 above)
Comment (phoracek) [reply manifests/cluster-network-addons/0.8.0/cluster-network-addons-operator.0.8.0.clusterserviceversion.yaml]: huh, we forgot to add it to templates, thanks

**Comment by phoracek** [comment]
@qinqon @SchSeba rebased on current master

**Pull request merged**